.circleci/workflows.yml

```diff
@@ -289,6 +289,10 @@ jobs:
             - checkout
             - *restore_venv_cache
             - *build
+            - run:
+                name: Validate workgroup access configuration on main
+                command: |
+                  PATH="venv/bin:$PATH" script/bqetl metadata validate-workgroups sql/
```

Comment:
Run the metadata workgroup access validation before generated SQL gets copied over. During SQL generation also metadata is generated, which updates and sets the `default_table_workgroup_access` and `workgroup_access` configs and would make the CI check fail.